client/src/components/global/SignInUp/PasswordForgot.js

```diff
@@ -1,28 +1,102 @@
 import React, { Component, PropTypes } from 'react';
+import { passwordsAPI } from 'api';
+import { entityStoreActions, notificationActions } from 'actions';
+import classNames from 'classnames';
+import { get } from 'lodash';
+import { connect } from 'react-redux';
+const { request, flush } = entityStoreActions;
 
-export default class PasswordForgot extends Component {
+class PasswordForgot extends Component {
 
   static propTypes = {
     showLogin: PropTypes.func.isRequired,
-    showCreate: PropTypes.func.isRequired
+    showCreate: PropTypes.func.isRequired,
+    hideSignInUpOverlay: PropTypes.func
   };
 
+  static mapStateToProps(state, ownProps) {
+    return {
+      response: get(state.entityStore.responses, 'request-reset-password')
+    };
+  }
+
+  constructor(props) {
+    super();
+    this.state = { email: '' };
+    this.handleInputChange = this.handleInputChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.submissionError = this.submissionError.bind(this);
+  }
+
+  componentWillUnmount() {
+    this.props.dispatch(flush(['request-reset-password']));
+  }
+
+  handleSubmit(event) {
+    event.preventDefault(event.target);
+    this.props.dispatch(request(passwordsAPI.new(this.state.email), 'request-reset-password')).promise.then(() => {
```

Comment:
Resetting pw is special and can't be handled as a generic API request using the request helper. We need to make an authorization action and reducer method to handle this, because after the request is resolved, login should occur. Look at how login works, and see if you can figure out how pw reset might work.